ddtrace/contrib/celery/task.py

```diff
@@ -87,7 +87,7 @@ def _task_run(pin, func, task, args, kwargs):
 
 @require_pin
 def _task_apply(pin, func, task, args, kwargs):
-    with pin.tracer.trace(TASK_APPLY, resource=task.name) as span:
+    with pin.tracer.trace(TASK_APPLY, service=pin.service, resource=task.name) as span:
```

Comment:
Usually the `apply()` is called by primary services (i.e. Django or Flask). If we omit this, it means that it would be part of the Django / Flask service. Not sure if it's OK because we may have an exception with the broker / job creation that would be considered a Django / Flask fail.